src/navigator/NavigatorWrapper.tsx

```diff
@@ -167,6 +167,7 @@ export const NavigatorWrapper = () => {
 
   return (
     <NavigationContainer
+      navigationInChildEnabled={true}
```

Comment:
```suggestion
      navigationInChildEnabled
```
I think this should work for boolean props